qa/tasks/cephfs/test_volumes.py

```diff
@@ -5212,6 +5212,230 @@ def test_clone_subvolume_info(self):
         # verify trash dir is clean
         self._wait_for_trash_empty()
 
+    def test_subvolume_snapshot_info_without_snapshot_clone(self):
+        """
+        Verify subvolume snapshot info output without clonnnig snapshot.
```

Comment:
cloning?

Reply:
@nmshelke I guess you missed this change!!